Automattic/wp-calypso
# Remove Anchor fm

Fixes https://github.com/Automattic/wp-calypso/issues/75387

Related to:
* https://github.com/Automattic/wp-calypso/pull/77406
* https://github.com/Automattic/wp-calypso/pull/78315
## Proposed Changes

This should be merged after [this PR](https://github.com/Automattic/wp-calypso/pull/79295).

Removes Anchor fm related files from stepper.

## Testing Instructions



* Look at the code
* Verify tests run properly
* Go through some stepper flows to verify it's working correctly

## Pre-merge Checklist
